pkgs/development/tools/devpi-server/default.nix

```diff
@@ -32,12 +32,17 @@ python3Packages.buildPythonApplication rec {
     webtest
   ] ++ stdenv.lib.optionals isPy27 [ mock ];
 
-  # test_genconfig.py needs devpi-server on PATH
   # root_passwd_hash tries to write to store
+  # TestMirrorIndexThings tries to write to /var through ngnix
+  # nginx tests try to write to /var
```

Comment:
that stuff is super creepy!

Reply:
nginx whats to open a `/var/log/nginx/error.log`, which kind of makes sense

Reply:
"kind of", haha. i am still trying to build your PR with `nix-review`, it takes some time